src/main/java/mate/academy/intro/model/Book.java

```diff
@@ -28,4 +32,6 @@ public class Book {
     private BigDecimal price;
     private String description;
     private String coverImage;
+    @Column(name = "is_deleted", nullable = false, columnDefinition = "TINYINT(1) DEFAULT 0")

```

Comment:
```suggestion
    @Column(nullable = false, columnDefinition = "TINYINT(1) DEFAULT 0")
```

Hibernate automatically create column in snake_case style